scripts/verifyProposal.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
+
+import { proposeVIP } from "../src/transactions";
+
+export async function getTransactionData(vip: number, txHash: string, networkId: number) {
+  // create an Ethers.js provider based on the network ID
+  const provider = ethers.getDefaultProvider(getNetworkName(networkId));
+  // get the transaction object using the provider and transaction hash
+  const tx = await provider.getTransaction(txHash);
+  const buildData = await proposeVIP(vip);
+  return buildData.calldata === tx["data"];
+}
```

Comment:
We should also have a script to verify multisig tx data _before_ the transaction is submitted to chain, e.g. when someone from the team proposes a VIP, and the other team member runs this script to validate it. But I assume it's out of the scope for now.

Reply:
i think it's already implemented.